### DISCUSSION TITLE
How much RAM does it require approximately?

### DISCUSSION THREAD

**Discussion by naddika** [jlelse/GoBlog#61] [General]
How much RAM does it require approximately?

**Comment by naddika** [comment]
17Mb? Or 1901Mb which is almost 2Gb?
The former appears to be too small, the latter - too large.

![Screenshot_20230813_094249](https://github.com/jlelse/GoBlog/assets/44301454/7a794b51-91f9-4b15-a799-3d7db122ce46)

**Comment by jlelse** [comment]
My instance (jlelse.blog) uses about 140 MB and I also have a few plugins and a lot of features enabled. The instance for goblog.app uses much less.
